src/Control/Monad/Qualified.purs

```diff
@@ -0,0 +1,42 @@
+-- | Use this module to rebind `do`/`ado` notation via Qualfieid Do/Ado.
```

Comment:
"Qualfieid" typo :wink:

Reply:
Whoops! Good catch. 

I've fixed it in this [PR's  branch](https://github.com/JordanMartinez/purescript-indexed-monad/commits/master), but the changes aren't being included in this PR for some reason.